docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -0,0 +1,108 @@
+[[searchable-snapshots]]
+== {search-snaps-cap}
+
+Nodes in a distributed system like {es} will inevitably fail from time to time.
+To protect your data against node failures, by default when you index a
+document into {es} it is stored on two or more nodes. You also take periodic
+<<snapshot-restore,snapshots>> of your data so that you can recover from more
+serious failures. This means that each document is stored in at least three
+places. These extra copies are important for resiliency, but the storage they
+consume has an impact on your cluster's operating costs. The two storage
+mechanisms have different, but complementary, performance characteristics:
+
+* Snapshot repositories are much more reliable than local storage on individual
+  nodes.
+
+* The monetary cost per GB stored in a snapshot repository is usually lower
+  than on a node.
+
+* The monetary cost per read or write operation on a snapshot repository is
+  usually much higher.
+
+* Reading or writing data in a snapshot repository usually takes much more time
+  compared with accessing a node's local storage.
+
+{search-snaps-cap} let you reduce your operating costs by treating the snapshot
+as the authoritative copy of some of your indices. The high reliability of the
+snapshot repository removes the need to keep multiple copies of their data in
+your cluster purely for resiliency. {es} makes a copy of a searchable snapshot
+on the nodes in the cluster to reduce the performance impact and costs of
+accessing the snapshot repository.
+
+With {search-snaps-cap} you may be able to halve your cluster size without
+increasing the risk of data loss or reducing the amount of data exposed to
+searches. Put differently, {search-snaps-cap} may allow you to expose twice as
+much data to searches for a given cluster size.
+
+=== Using searchable snapshots
+
+A searchable snapshot can be searched just like any other index.
+{search-snaps-cap} are often used to access a large archive of historical data,
+for which searches may sometimes be complex and time-consuming.
+<<async-search>> is particularly useful for these long-running searches.
+
+The shards of searchable snapshots are also allocated just like shards of any
+other index. You can, for instance, use <<shard-allocation-filtering>> to
+restrict these shards to a subset of your nodes.
+
+Normally you will use {search-snaps-cap} via the
```

Comment:
"use" implies "search" in my head, I would prefer to say "create" or "mount" here?

Reply:
I wanted to talk about something more general than just creating them -- after all ILM does also take care of aliases which lets you search them too. How about `manage`? 90a2184f94fb078b0db8858fd593b4829a349266